### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by SerhiiKondrashenko**
Finished Task 2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
```
Comment (hamanovich) [inline package.json]: what the diff between this line and line #10, 'client:deploy' command?

Comment (SerhiiKondrashenko) [reply package.json]: Hi! Seems there is no diff between them, they are do the same. I used this one, but I also ok with [rolling-scopes-school#10](https://github.com/rolling-scopes-school/nodejs-aws-fe/pull/10). Is it incorrect or anything else?

Comment (SerhiiKondrashenko) [reply package.json]: When I've been doing the task I had little understanding what sls deploy does, so I wrote my own command.

Comment (MikeShakmakov) [reply package.json]: sls is just alias for serverless so its duplicated command

**Review by MikeShakmakov — APPROVED** [review]
the code looks fine(the only small issue is duplicated command)

File: package.json (hunk 1 above)
Comment (SerhiiKondrashenko) [reply package.json]: Ok, I have removed duplicated scripts, ty for suggestions.

**Pull request merged**